scss/login.scss

```
@@ -0,0 +1,57 @@
+.login-container {
+  margin: 100px auto;
+  width: fit-content;
+}
+
+.login-form {
+  background: $wild-sand;
```

Don't use shorthands for only one property 
```suggestion
  background-color: $wild-sand;
```